esp/src/src/nls/hpcc.ts

```diff
@@ -639,6 +641,7 @@ export = {
         PlaceholderFirstName: "John",
         PlaceholderLastName: "Smith",
         Platform: "Platform",
+        PlatformIsNNNDaysOld: "Platform is NNN days old",
```

Comment:
Suggest reword to 
"Platform build is NNN days old."

Once it is reported as a warning it should be

"Platform build is NNN days old.  Please upgrade to a later point release."

to make it clear that it isn't the version we are complaining about, and how to resolve it.

Reply:
Possibly add "(See About/Latest Releases for details.)"